### DISCUSSION TITLE
I need to erase dark vertical stripe in center, tile, and normalize margins in 206 double-page scans

### DISCUSSION THREAD

**Discussion by oavaldezi** [ImageMagick/ImageMagick#2010] [General] [answered]
I have 206 images from a book that was laid out flat two pages at a time on the scanner. I need to 1) erase the dark vertical stripe of variable width in the center of all images, 2) split them into two horizontal tiles (I know I can do that with '-crop 2x1@ +repage', and 3) "normalize" the margins on the split tiles (similar left and right margins on left and right tiles).
Below is a screenshot of some of the images, and I'm also attaching a couple of them. Thanks in advance for the help.

![imagen](https://user-images.githubusercontent.com/2160804/81755754-dd6a3b80-9476-11ea-91df-b07d0adf179b.png)
![imagen-0022](https://user-images.githubusercontent.com/2160804/81756290-941aeb80-9478-11ea-88ac-ae550493d781.jpg)
![imagen-0023](https://user-images.githubusercontent.com/2160804/81756303-98df9f80-9478-11ea-95b6-403a01fd8c0d.jpg)

**Comment by snibgo** [comment]
If the central dark gutter is always in roughly the same position, you can "paint" it out with white. Then chop the image into two pieces, trim the images, and add whatever border or "-extent" you want. For example, Windows BAT syntax:

[code]magick ^
  81756290-941aeb80-9478-11ea-88ac-ae550493d781.jpg ^
  -fill White -draw "rectangle 1410,0,1600,%%[fx:h-1]" ^
  -crop 2x1@ +repage ^
  -trim ^
  -bordercolor White -border 50 ^
  out.png[/code]

**Comment by oavaldezi** [comment]
Thanks! It works with PNG and JPG output, but not with TIFF: the output is garbled, and according to GIMP. it has a gray and an alpha channel. But if I don't draw the rectangle, the TIFFs have only one 'indexed' channel, and the images are what is expected. How can I fix this?
Can't attach TIFFs directly, so I've zipped two of them.
BTW, there's only one '%' before [fx:h-1]: `-fill White -draw "rectangle 1410,0,1600,%[fx:h-1]"`
[images.zip](https://github.com/ImageMagick/ImageMagick/files/4622267/images.zip)

**Comment by snibgo** [comment]
I showed a command with BAT syntax. This needs doubled %%. If you run it from the console command line, don't double the %.

We should "+repage" after the "-trim". I forgot that, sorry.

Gimp 2.10.0 can read the tiffs when I run the command, and when I use your uploaded imagen016.tiff and imagen019.tiff. The images do not seem to be garbled.

**Reply by oavaldezi** [reply]
It may have sent out the wrong files. I'm attaching a couple of garbled samples. Do those display properly in your GIMP?
But I fixed the issue by adding `-alpha off ` before cropping! Thanks again!
[images2.zip](https://github.com/ImageMagick/ImageMagick/files/4622961/images2.zip)

**Comment by snibgo** [comment, marked as answer]
imagen000.tiff and imagen001.tiff have 2 bits per pixel; one bit for black/white and one bit for alpha. Gimp and Windows Photo Viewer show garbage. I don't know if this is a problem in IM, or in Gimp and Photo Viewer.

A solution is "-alpha off", as you say.
